src/_guides/language/effective-dart/usage.md

```diff
@@ -371,28 +372,27 @@ class Circle {
   }
 
   void _recalculate() {
-    _area = math.PI * _radius * _radius,
+    _area = math.PI * _radius * _radius;
     _circumference = math.PI * 2.0 * _radius;
   }
 }
 {% endprettify %}
-</div>
 
 That's an awful lot of code to write, maintain, debug, and read. Instead, your
 first implementation should be:
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (cacl-vs-store)"?>
 {% prettify dart %}
 class Circle {
   num radius;
 
+  Circle(this.radius);
+
   num get area => math.PI * radius * radius;
   num get circumference => math.PI * 2.0 * radius;
-
-  Circle(this.radius);
```

Comment:
I moved the constructor so as to follow our own guidelines :) ... otherwise the analyzer (w/ lint rules enabled) complains.

Reply:
But it's OK to have `num radius` above? It seems weird to split the property declarations.

Reply:
The rule is [sort_constructors_first](http://dart-lang.github.io/linter/lints/sort_constructors_first.html):

> DO sort constructor declarations before method declarations.

So field declarations ahead of the constructor are ok. I sometimes feel that getters and setters seem like they should be on par with fields, but they are not.